packages/rancher-windows-exporter/charts/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+version: 0.1.0
+appVersion: 0.0.4
+annotations:
+  catalog.rancher.io/certified: rancher
+  catalog.rancher.io/namespace: cattle-monitoring-system
```

Comment:
Should we be adding request cpu and request memory to all charts?

Reply:
Hmm I don't think we need this here since this chart is never expected to be installed without Monitoring V2 (it will fail since the CRDs aren't installed, similar to `rancher-pushprox`). The CPU / Request memory annotations in Monitoring V2 should be sufficient.

@cbron agree?

Reply:
Yes cpu/mem only at top level chart, unless chart can be deployed separately. It would be cool to define at all levels and aggregate that back up, but not practical.